lib/Connection.js

```diff
@@ -265,6 +266,7 @@ class Connection {
   }
 
   toJSON() {
+    // eslint-disable-next-line no-unused-vars
     const { authorization, ...headers } = this.headers;
```

Comment:
Note to reviewer: Even though `authorization` is not being used, its declaration is still necessary to remove `authorization` from `headers`